src/command_modules/azure-cli-iot/HISTORY.rst

```diff
@@ -5,6 +5,9 @@ Release History
 
 0.1.17
 ++++++
+* iot dps access policy: Certain commands are now long-running operations.
+* iot dps linked-hub: Certain commands are now long-running operations.
```

Comment:
These were already LRO. From our discussion it seems like the text should be something like:

"* `iot dps create/update`: These commands now return the credentials upon completion."

Reply:
These commands would return credentials before I changed them but often return wrong credentials as I didn't wait for the create_and_update operations. So I guess, it is still changed to long-running? I will add --no-wait support for description.